workflow-templates/python_build.yml

```diff
@@ -0,0 +1,64 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python on Ubuntu.
+
+name: Python build - linting and testing
+
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
```

Comment:
Add a step to checkout git submodules.
https://github.com/padaiyal/.github/blob/5c73b3bbec9a07313241e586439e8d923160a1ce/workflow-templates/maven_build.yml#L22